proto/teleport/lib/teleterm/v1/windows_desktop.proto

```diff
@@ -0,0 +1,40 @@
+/*
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+syntax = "proto3";
+
+package teleport.lib.teleterm.v1;
+
+import "teleport/lib/teleterm/v1/label.proto";
+
+option go_package = "github.com/gravitational/teleport/gen/proto/go/teleport/lib/teleterm/v1;teletermv1";
+
+// WindowsDesktop describes a Windows desktop resource.
+message WindowsDesktop {
+  // uri uniquely identifies the Windows desktop within Teleport Connect.
+  string uri = 1;
+  // name is the name of the desktop.
+  string name = 2;
+  // address of the desktop.
+  // Omits the port if it's the default RDP port (3389).
```

Comment:
Feel free to adjust it in another PR, but wouldn't it make sense for tshd to always return the full address and omit the port only before displaying the address in the UI? If any part of the Electron app ever needs the port, we'll have to introduce a breaking change.

This doesn't matter that much now, but I'm thinking of a situation where the menu bar app and Connect share the same tsh daemon instance for example.

Reply:
Yeah, I think you're right. I did it here mainly because Web UI strips the port in the API handler. Changed.